Tests/SonarScanner.MSBuild.PreProcessor.Test/TargetsInstallerTests.cs

```diff
@@ -88,7 +88,11 @@ public void InstallTargetsFile_Overwrite()
         var msBuildPathSettings = new MsBuildPathSettings();
 
         InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
-        msBuildPathSettings.GetImportBeforePaths().Should().HaveCount(7, "Expecting six destination directories");
+        // If the current user account is LocalSystem, then the local application data folder is inside %windir%\system32.
+        // When a 32-bit process tries to use this folder on a 64-bit machine, it is redirected to %windir%\SysWOW64.
+        // In that case the scanner needs to deploy ImportBefore.targets to both locations, doubling the number of destination directories (14 instead of 7).
+        var hasCorrectCount = msBuildPathSettings.GetImportBeforePaths().Count() is 7 or 14;
+        hasCorrectCount.Should().BeTrue("Expecting 7 destination directories (or 14 if the local application data folder is located in %windir%)");
```

Comment:
Why was this not a problem before?

Reply:
The builds on the previous image were running under a different user account (`VssAdmin`). The local app data folder for this account was not in %windir%, so there was only one of it.
With the new image it's now running under the `LocalSystem` user account. The new folders are:

```
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\4.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\10.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\11.0\Microsoft.Common.targets\ImportBefore", 
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\12.0\Microsoft.Common.targets\ImportBefore", 
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\14.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\15.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\system32\config\systemprofile\AppData\Local\Microsoft\MSBuild\Current\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\4.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\10.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\11.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\12.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\14.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\15.0\Microsoft.Common.targets\ImportBefore",
"C:\Windows\SysWOW64\config\systemprofile\AppData\Local\Microsoft\MSBuild\Current\Microsoft.Common.targets\ImportBefore"
```